### PR TITLE
add description about insufficient length problem for mysql , mariadb

### PR DESCRIPTION
**Pull request by charlietag**
Both MySQL and MariaDB have the same problem.  So I add some note into the document.  Hope it's useful to remind user, that it could be happened and how to solve it through #110 .

### CODE REVIEW

**Comment by shioyama** [comment]
Thanks! Will have a look and merge this very soon.

File: README.md [hunk 1]
```diff
@@ -102,6 +102,12 @@ end
 To use a different default backend, set `default_backend` to another value (see
 possibilities [below](#backends)).
 
+If you choose `key_value` as `default_backend`, and **MySQL / MariaDB** is your
```
Comment (shioyama) [inline README.md:105]: The `default_backend` config has changed on the master branch (for v1.0). Could you reword this to:

> If you use `key_value` as your backend, ...

This way the meaning is more general, and I can also cherry-pick this into the 0-8-stable branch.

Comment (charlietag) [reply README.md]: @shioyama 
Thanks for your suggestion. I've pushed a new commit for this.

**Comment by shioyama** [comment]
Hi @charlietag, sorry for the delay. I'm planning to rewrite the readme, so this won't fit very well right now. I'll cherry-pick it in when I figure out how things will be arranged.

**Comment by charlietag** [comment]
Hi @shioyama , that's ok.
Thanks for your effort.
So I can use this so convenient gem!
